presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveSplit.java

```diff
@@ -55,6 +55,10 @@ public void testJsonRoundTrip()
         AcidInfo.Builder acidInfoBuilder = AcidInfo.builder(new Path("file:///data/fullacid"));
         acidInfoBuilder.addDeleteDelta(new Path("file:///data/fullacid/delete_delta_0000004_0000004_0000"), 4L, 4L, 0);
         acidInfoBuilder.addDeleteDelta(new Path("file:///data/fullacid/delete_delta_0000007_0000007_0000"), 7L, 7L, 0);
+
+        acidInfoBuilder.addOriginalFile(new Path("file:///data/fullacid/000000_0"), 120, 0);
+        acidInfoBuilder.addOriginalFile(new Path("file:///data/fullacid/000001_0"), 125, 0);
```

Comment:
Redundant?

Reply:
We need this to compare the acidInfo.

Reply:
This doesn't affect the test flow, so i do not understand why it's being added here. Let me remove.